CHANGELOG.md

```diff
@@ -8,6 +8,15 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## Unreleased
 
+### Added
+
+### Changed
+- Uses Prometheus `/api/v1/status/config` endpoint to read the Prometheus 
+  config, to automatically determine the full set of scrape intervals. (#162)
```

Comment:
I think I'll make the command-line a no-op so that users are not impacted. I also need to check how far back this web API goes.